runners/console/index.ts

```diff
@@ -93,6 +93,58 @@ export class Console extends Runner {
         return this.runInstallDevonfwIde(runCommand);
     }
 
+    runRestoreWorkspace(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let workspacesName = "workspace-" + ((runCommand.command.parameters.length > 0 && runCommand.command.parameters[0].workspace)
+            ? runCommand.command.parameters[0].workspace
+            : this.playbookName.replace("/", "").replace(" ","-"));
+
+        let workspacesDir = this.getVariable(this.useDevonCommand)
+            ? path.join(this.getWorkingDirectory(), "devonfw", "workspaces")
+            : this.getVariable(this.workspaceDirectory);
+
+        //removes all the directories and files inside workspace
+        if(this.getVariable(this.useDevonCommand)){
+            fs.rmdirSync(path.join(workspacesDir), { recursive: true });
+            fs.mkdirSync(workspacesDir);
+        }
+
+        //copies a local repository into the workspace
+        if(runCommand.command.parameters.length > 0 && runCommand.command.parameters[0].local){
+            let forkedWorkspacesDir = path.join(this.getWorkingDirectory(),'..','..','..', workspacesName);
+            if(fs.existsSync(forkedWorkspacesDir))
+                fs.copySync(path.join(forkedWorkspacesDir, '/.',), workspacesDir)  
```

Comment:
Is the '/.' necessary here?
And I think there is a comma to much the end of path.join()

Reply:
yes, otherwise it will copy the whole folder into workspacesDir. In this case I just want to copy the content of the folder into the workspaces directory